### ISSUE TITLE
Exclude dev site traffic from analytics

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/Coordination#622]

**Comment by PatReynolds** [comment]
Perhaps manage with google tag manager - revisit after gtm implemented

**Comment by richpomfret** [comment]
This is a nice idea, however dev site traffic really should be so minor that I doubt it will have much influence over our GA stats.

**Comment by DeniseColbert** [comment]
Agreed, but it although traffic is low, the number of Admin pages logged is high so it makes for messy viewing. It's not a big task to filter it out. Worth mentioning that this story was created with just FreeUKG site in mind.
